components/tag/TagActionButton.vue

```diff
@@ -12,10 +12,20 @@ const emit = defineEmits<{
 const { client } = $(useMasto())
 
 const toggleFollowTag = async () => {
-  if (tag.following)
+  if (tag.following) {
     await client.v1.tags.unfollow(tag.name)
-  else
+      .then(() => {
+        // eslint-disable-next-line vue/no-mutating-props
```

Comment:
elsewhere we destructure a value from the prop so we don't mutate the value...

Reply:
For now this is okay, as it would only affect this for a short time until you reload the context or whatever

I plan in a future PR to build up a cache like `cacheStatus` for tags
This way we can also (maybe) save some api-calls when accessing a tag page a second time, but I need to try that